packages/bierzo-wallet/src/logic/csvBuilder/index.ts

```diff
@@ -0,0 +1,52 @@
+export interface CsvRow {
+  readonly id: string;
+  readonly recepient: string;
+  readonly sender: string;
+  readonly quantity: string;
+  readonly fractionalDigits: string;
+  readonly tokenTicker: string;
+  readonly feeQuantity: string;
+  readonly feeFractionalDigits: string;
+  readonly feeTokenTicker: string;
+  readonly time: string;
+  readonly note: string;
+  [key: string]: string;
+}
+
+class CsvRepresentation {
+  private readonly _fields: readonly string[];
+  private readonly _header: string;
+  private readonly _rows: CsvRow[] = [];
```

Comment:
Why the `_ ` prefixes? We don't use them in the rest of the project as far as I know